examples/svelte-kit/vite.config.ts

```diff
@@ -0,0 +1,13 @@
+import { sveltekit } from '@sveltejs/kit/vite'
+import type { UserConfig } from 'vite'
+import { telefunc } from 'telefunc/vite'
+
+const config: UserConfig = {
+  plugins: [
+    // FIXME: telefunc dev server middleware conflicts with sveltekit `src/routes/_telefunc/+server.ts`
+    telefunc().filter((p: any) => p.name !== 'vite-plugin-ssr:devConfig:serverMiddleware'),
```

Comment:
@brillout not sure how I'm suppose to handle this

Reply:
Actually, thinking of it, automatically adding `telefunc()` to Vite's dev and preview server doesn't make much sense. (Because in production the user will always have a Node.js/Edge server, so he'll need to manually integrate `telefunc()` with his server anyways.)

Consequently, I think we should remove this auto install thing, or at least make it opt-in. I'll think about it.

Reply:
I will remove auto install later today / tomorrow.

Reply:
I didn't remove the auto install but I made it run after SvelteKit so it doesn't conflict, see https://github.com/brillout/telefunc/pull/54/commits/21998d32c3227f2f90eeb7638b027f0e3dd10fb9.